src/animators/baseAnimator.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * Copyright 2017-present Palantir Technologies, Inc. All rights reserved.
```

Comment:
(probably a separate PR) you could use the `file-header` lint rule to enforce this